docs/monitors/collectd-custom.md

```diff
@@ -53,6 +53,24 @@ directory, where `<AGENT_BUNDLE>` is the root of the filesystem in the
 containerized version, and is normally `/usr/lib/signalfx-agent` in the
 non-containerized agent.
 
+## Running the collectd/exec plugin
+You can use the collectd/custom monitor to run the collectd/exec plugin.
+If you are not running the agent in the container, you can use any appropriate user
```

Comment:
If you are not running the Smart Agent agent in the container, then you can use any appropriate user on your system.